server/events.go

```diff
@@ -1681,6 +1727,25 @@ func (s *Server) sendAccConnsUpdate(a *Account, subj ...string) {
 	a.mu.Unlock()
 }
 
+// Lock shoulc be held on entry
+func (a *Account) accConns() *AccountStat {
```

Comment:
Since this is an `Account`, I would rename `accConns()` - since we know it is an account - to either conns() or connStats() or something like that.

Reply:
renamed to `statz` as to not overlap with `stats` inside the account